### PR TITLE
Terminology updates in line with technical paper #2 Fantom Framework

### PR DESCRIPTION
**Pull request by andrecronje**
Terminology changes to reflect technical paper #2, terminology only, no implementation changes yet.

### CODE REVIEW

File: src/poset/poset_test.go [hunk 1]
```diff
@@ -407,14 +407,14 @@ func TestSee(t *testing.T) {
 	}
 
 	for _, exp := range expected {
-		a, err := p.see(index[exp.descendant], index[exp.ancestor])
+		a, err := p.dominatedindex[exp.dominated], index[exp.dominator])
```
Comment (dzeckelev) [inline src/poset/poset_test.go]: `a, err := p.dominated(index[exp.dominated], index[exp.dominator])`

File: src/poset/badger_store_test.go [hunk 2]
```diff
@@ -286,14 +286,14 @@ func TestDBRoundMethods(t *testing.T) {
 		t.Fatalf("Round and StoredRound do not match")
 	}
 
-	witnesses := store.RoundWitnesses(0)
-	expectedWitnesses := round.Witnesses()
-	if len(witnesses) != len(expectedWitnesses) {
-		t.Fatalf("There should be %d witnesses, not %d", len(expectedWitnesses), len(witnesses))
+	clothos := store.RoundClothos(0)
```
Comment (dzeckelev) [inline src/poset/badger_store_test.go]: `store.RoundClothos` method does not exist.

File: src/poset/badger_store_test.go [hunk 3]
```diff
@@ -286,14 +286,14 @@ func TestDBRoundMethods(t *testing.T) {
...
+	expectedClothos := round.Clothos()
```
Comment (dzeckelev) [inline src/poset/badger_store_test.go]: `round.Clothos` method does not exist.

File: src/poset/poset_test.go [hunk 4]
```diff
@@ -729,38 +729,38 @@ func TestStronglySee(t *testing.T) {
...
-	round0Witnesses := make(map[string]*RoundEvent)
-	round0Witnesses[index[e0]] = &RoundEvent{
-		Witness: true, Famous: Trilean_UNDEFINED}
-	round0Witnesses[index[e1]] = &RoundEvent{
-		Witness: true, Famous: Trilean_UNDEFINED}
-	round0Witnesses[index[e2]] = &RoundEvent{
-		Witness: true, Famous: Trilean_UNDEFINED}
+	round0Clotho := make(map[string]*RoundEvent)
+	round0Clotho[index[e0]] = &RoundEvent{
+		Clotho: true, Famous: Trilean_UNDEFINED}
```
Comment (dzeckelev) [inline src/poset/poset_test.go]: Famous -> Atropos

**Comment by andrecronje** [comment]
@dzeckelev sorry for wasting time by not running tests :'(

File: src/poset/badger_store.go [hunk 5]
```diff
@@ -265,12 +265,12 @@ func (s *BadgerStore) LastRound() int64 {
 	return s.inmemStore.LastRound()
 }
 
-func (s *BadgerStore) RoundWitnesses(r int64) []string {
+func (s *BadgerStore) RoundClotho(r int64) []string {
```
Comment (devintegral) [inline src/poset/badger_store.go]: RoundClotho**s**

File: src/poset/caches.go [hunk 6]
```diff
@@ -19,7 +19,7 @@ func (k Key) ToString() string {
 type ParentRoundInfo struct {
 	round                     int
 	isRoot                    bool
-	rootStronglySeenWitnesses int
+	rootStronglySeenClotho int
```
Comment (devintegral) [inline src/poset/caches.go]: rootStronglySeenClotho**s**

File: src/poset/inmem_store.go [hunk 7]
```diff
@@ -251,12 +251,12 @@ func (s *InmemStore) LastRound() int64 {
 	return s.lastRound
 }
 
-func (s *InmemStore) RoundWitnesses(r int64) []string {
+func (s *InmemStore) RoundClotho(r int64) []string {
```
Comment (devintegral) [inline src/poset/inmem_store.go]: RoundClotho**s**

File: src/poset/roundInfo.go [hunk 8]
```diff
@@ -39,36 +39,36 @@ func (r *RoundInfo) SetConsensusEvent(x string) {
...
 			return false
 		}
 	}
 	return true
 }
 
-//return witnesses
-func (r *RoundInfo) Witnesses() []string {
+//return clothos
+func (r *RoundInfo) Clotho() []string {
```
Comment (devintegral) [inline src/poset/roundInfo.go]: Clotho**s**()

File: src/poset/roundInfo.go [hunk 9]
```diff
@@ -96,20 +96,20 @@ func (r *RoundInfo) ConsensusEvents() []string {
 	return res
 }
 
-//return famous witnesses
-func (r *RoundInfo) FamousWitnesses() []string {
+//return Atropos
+func (r *RoundInfo) Atropos() []string {
```
Comment (devintegral) [inline src/poset/roundInfo.go]: Atropos**es**()

File: src/poset/store.go [hunk 10]
```diff
@@ -23,7 +23,7 @@ type Store interface {
 	GetRound(int64) (RoundInfo, error)
 	SetRound(int64, RoundInfo) error
 	LastRound() int64
-	RoundWitnesses(int64) []string
+	RoundClotho(int64) []string
```
Comment (devintegral) [inline src/poset/store.go]: RoundClotho**s**

File: src/poset/store_debug.go [hunk 11]
```diff
@@ -23,7 +23,7 @@ type Store interface {
 	GetRound(int64) (RoundInfo, error)
 	SetRound(int64, RoundInfo) error
 	LastRound() int64
-	RoundWitnesses(int64) []string
+	RoundClotho(int64) []string
```
Comment (devintegral) [inline src/poset/store_debug.go]: RoundClotho**s**

File: src/service/service.go [hunk 12]
```diff
@@ -32,13 +32,13 @@ func (s *Service) Serve() {
...
 	mux.Handle("/participants/", corsHandler(s.GetParticipants))
-	mux.Handle("/event/", corsHandler(s.GetEvent))
+	mux.Handle("/event/", corsHandler(s.GetEventBlock))
 	mux.Handle("/lasteventfrom/", corsHandler(s.GetLastEventFrom))
 	mux.Handle("/events/", corsHandler(s.GetKnownEvents))
 	mux.Handle("/consensusevents/", corsHandler(s.GetConsensusEvents))
 	mux.Handle("/round/", corsHandler(s.GetRound))
 	mux.Handle("/lastround/", corsHandler(s.GetLastRound))
-	mux.Handle("/roundwitnesses/", corsHandler(s.GetRoundWitnesses))
+	mux.Handle("/roundclothos/", corsHandler(s.GetRoundClotho))
```
Comment (devintegral) [inline src/service/service.go]: s.GetRoundClotho**s**

**Pull request merged**